### DISCUSSION TITLE
WARNING  No labels found

### DISCUSSION THREAD

**Discussion by Harshit28j** [ultralytics/yolov5#12187] [Q&A]
Hey!, I'm New to this, I don't understand why I'm getting this error.

```
WARNING  No labels found in C:\Users\Harsh\OneDrive\Desktop\yolov5-master\data\labels\train.cache, training may not work correctly.
```

when trying to run this code.

```
from ultralytics import YOLO
model=YOLO("yolov8n.yaml")  #build a new model from scratch
results=model.train(data=r"C:\Users\Harsh\OneDrive\Desktop\yolov5-master\data\custom_file.yaml", epochs=1) #train the model
```

I would truly appreciate your insights and guidance. Feel free to share
